backend/src/middleware/user/userMiddleware.js

```diff
@@ -1,16 +0,0 @@
-import createOrUpdateLocations from '../nodes/locations'
-
-export default {
-  Mutation: {
-    SignupVerification: async (resolve, root, args, context, info) => {
-      const result = await resolve(root, args, context, info)
-      await createOrUpdateLocations(result.id, args.locationName, context.driver)
-      return result
-    },
-    UpdateUser: async (resolve, root, args, context, info) => {
-      const result = await resolve(root, args, context, info)
-      await createOrUpdateLocations(args.id, args.locationName, context.driver)
-      return result
-    },
-  },
-}
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-19T14:32:26Z" title="Thursday, December 19th 2019, 3:32:26 pm +01:00">Dec 19, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -1,16 +0,0 @@
-import createOrUpdateLocations from '../nodes/locations'
-
-export default {
-  Mutation: {
-    SignupVerification: async (resolve, root, args, context, info) => {
-      const result = await resolve(root, args, context, info)
-      await createOrUpdateLocations(result.id, args.locationName, context.driver)
-      return result
-    },
-    UpdateUser: async (resolve, root, args, context, info) => {
-      const result = await resolve(root, args, context, info)
-      await createOrUpdateLocations(args.id, args.locationName, context.driver)
-      return result
-    },
-  },
-}
```


why are these changes in this `PR` @roschaefer 
code creep?